utils/config.ts

```diff
@@ -90,15 +107,28 @@ export const IConfig = t.intersection([
     NOTIFICATION_QUEUE_NAME: NonEmptyString,
     NOTIFICATION_QUEUE_STORAGE_CONNECTION_STRING: NonEmptyString,
 
+    RC_CONFIGURATION_CACHE_TTL: NonNegativeIntegerFromString,
```

Comment:
if we put this in the base config we need to add it to the helm env variables. Instead we can define it with a default in errorOrConfig.

Reply:
[47812ee](https://github.com/pagopa/io-functions-service-messages/pull/57/commits/47812ee86a26df6181c61fcb084a614c3c0f8e72)